xamarin/Xamarin.Forms
# [Android] Fixed SwipeView child interaction issue on Android

### Description of Change ###

We can use a `SwipeItemView` that allows a View as content. On **Android**, the `SwipeView` is intercepting the touch events of the `SwipeItemView` child elements.
This PR applies changes to fix this issue.

### Issues

- fixes #9007

### API Changes ###
 
 None

### Platforms Affected ### 

- Android

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

#### Before
![interaction-before](https://user-images.githubusercontent.com/6755973/69552473-cefa9700-0f9e-11ea-929a-a4b128b17a3e.gif)

#### After
![interaction-after](https://user-images.githubusercontent.com/6755973/69552478-d0c45a80-0f9e-11ea-9d0b-10420ec34643.gif)

### Testing Procedure ###
Launch Core Gallery and navigate to the SwipeView samples gallery. Select the Custom SwipeItems Gallery and try to to press the button from a Custom SwipeItem. If an alert appears, the test has passed (the event click of the button has been invoked).

### PR Checklist ###
